lib/browser/init.ts

```diff
@@ -186,6 +186,7 @@ function currentPlatformSupportsAppIndicator () {
 }
 
 // Workaround for electron/electron#5050 and electron/electron#9046
+process.env.ORIGINAL_XDG_CURRENT_DESKTOP = process.env.XDG_CURRENT_DESKTOP;
```

Comment:
Should this be documented ?

Reply:
Probably, but not sure where 🤔

Reply:
https://github.com/electron/electron/blob/master/docs/api/environment-variables.md would be good spot.

Reply:
Good call, done 👍